frontend/public/components/overview.jsx

```diff
@@ -135,24 +142,37 @@ class OverviewDetails extends React.Component {
     });
   }
 
-  groupByLabel(items, label) {
-    const groups = _.groupBy(items, (item) => _.get(item, ['metadata', 'labels', label], 'other'));
+  groupItems(items, label) {
+    const compareGroups = (a, b) => {
+      if (a.name === 'other') {
```

Comment:
This could potentially collide with a value since "other" is a legal label value

Reply:
This issue goes beyond just this sorting function. In the case where a resource has a label value of 'other', it would be grouped with resources which don't have a value for the currently selected 'group by' label. Since labels can contain almost any string value, isn't the possibility of collision always going to be an issue?

Reply:
Yeah, that would work to group them into a guaranteed unique group, but we would still have a problem with what to display as the heading for that group since we don't want that to collide either. We could use a value with a space in it which would make it an invalid label value, and it could still be used as a display value so that we don't have to do special logic when rendering. "No value" or something like that.

Reply:
Let's look at it in a follow on

Reply:
https://jira.coreos.com/browse/CONSOLE-755